fastlane/Fastfile

```diff
@@ -50,19 +54,22 @@ lane :upload_release do
 end
 
 desc 'Download the development signing certificates to this machine'
-lane :set_up_signing do
+lane :set_up_signing do |readonly: true|
```

Comment:
Injecting `readonly` allows us to run this lane locally to regenerate the code signing credentials.